rfcs/checkbox-component.md

```diff
@@ -126,7 +126,7 @@ export default () => {
 ## `StatefulCheckboxContainer` API
 
 * `initialState: {}`
-  Initial state of an uncontrolled popover component.
+  Initial state of an uncontrolled checkbox component.
```

Comment:
should we fix it a new PR? it seems to be out of context of this PR

Reply:
ok